### DISCUSSION TITLE
Media keys not working

### DISCUSSION THREAD

**Discussion by Pedramdx** [valinet/ExplorerPatcher#325] [General]
after some minutes the media keys will stop working and just volume keys are working

**Comment by Poopooracoocoo** [comment]
Are you certain that you cannot reproduce this without ExplorerPatcher?

**Comment by LiTO773** [comment]
Same issue here. When I use the Windows 10 taskbar the previous, next, and play/pause keys on my keyboard don't work. However, when I change back to the Windows 11 taskbar they work again.

**Reply by valinet** [reply]
Well, yeah, idk, it seem to work for me. I use play/pause all the time. Works in Firefox, Chrome, Microsoft Edge and VLC media player. I don't think it is an issue with ExplorerPatcher. I will play with out more, yet I don't touch anything related to that...
